src/views/Layout.jsx

```diff
@@ -16,24 +33,40 @@ export function Layout() {
 							<h1>Smart shopping list</h1>
 						</a>
 
-						<div>
-							<span>Welcome, {auth.currentUser.displayName}</span>
-						</div>
-						<SignOutButton />
+						{!isMobile && (
+							<div>
+								<div>
+									<span>Welcome, {auth.currentUser.displayName}</span>
+								</div>
+								<SignOutButton />
+							</div>
+						)}
 					</header>
 					<main className="Layout-main">
 						<Outlet />
 					</main>
+					{isMobile && (
+						<div className="mobileLogFooter">
+							<SignOutButton />
+						</div>
+					)}
 				</>
 			) : (
 				<>
 					<header className="Layout-header">
 						<h1>Smart shopping list</h1>
-						<div>
-							<SignInButton />
-						</div>
+						{!isMobile && (
+							<div>
+								<SignInButton />
+							</div>
+						)}
 					</header>
 					<LoggedOut />
+					{isMobile && (
+						<div className="mobileLogFooter">
+							<SignInButton />
+						</div>
```

Comment:
This button is the only one that will span fully across the screen, so I figured it'd be easier to just expand a div and fill it, since you can't add classNames to button components. I suppose you could also just make a custom button as we did for the squares on listView...But this way seemed more simplistic than defining a whole new set of css rules

Reply:
There's a couple ways we could approach this. 

- We could send width into the Button component props, and avoid having to make custom buttons at all. 
- You can add classNames to Button components (it already is on the Button component), it just needs to be sent in as a prop.

I think generally, it's best practice to avoid adding divs that aren't necessary, but someone correct me if I'm wrong!

Reply:
Ah for some reason, I thought we had struggled with setting button width width last night for some reason or another.

But yeah, if the component can do it, may as well use the component!